### ISSUE TITLE
Error message incorrectly displayed

### ISSUE THREAD

**Issue by huttonr** [huttonr/bootstrap3#14]
There is an error currently displayed in the console when using `1.3-beta.11`.

It displays: `Bootstrap disabled. Create a file named 'bootstrap-settings.json' to enable.`

This is a false-positive as Bootstrap is in-fact being included correctly so the detection is what isn't working correctly.

**Comment by sandervanhooft** [comment]
+1.
Bootstrap loads perfectly it seems. But I get the error message mentioned above. I am using this config so I can import Bootstrap in my other LESS files:

```
...
    "customVariables": false,

    "exposeMixins": false,

    "compile": false,
...
```

**Comment by huttonr** [comment]
This is now fixed in huttonr:bootstrap3@3.3.6_8.

The cause was https://github.com/meteor/meteor/issues/6409.
